daemon/job/active.go

```diff
@@ -496,6 +506,19 @@ func (j *ActiveSide) do(ctx context.Context) {
 		replicationReport := j.tasks.replicationReport()
 		j.promReplicationErrors.Set(float64(replicationReport.GetFailedFilesystemsCountInLatestAttempt()))
 
+		for _, attempt := range replicationReport.Attempts {
+			for _, filesystem := range attempt.Filesystems {
+				if filesystem.Info == nil {
```

Comment:
The "proper" way to check this is to inspect `attempt.State`, like so:

https://github.com/zrepl/zrepl/blob/83fdffbceff9d1c8b1690dc104f9cdbb7b2f8b87/replication/report/replication_report.go#L177-L191